### ISSUE TITLE
Is the registry now normative? 

### ISSUE THREAD

**Issue by sesuncedu** [w3c/microdata-rdf#12]
There seem to be dependencies on matching entries to determine prefixes that _are_ normative. 

Also, is matching against the registry by longest match, or may no entry be a prefix of another?

**Comment by gkellogg** [comment]
You're correct that the algorithm is ambiguous if there are multiple entries in the registry which are a prefix of the `@itemtype` value. This doesn't really affect anything now, as the published registry (existing and soon-to-be-updated) contains no such ambiguities. It would be possible to construct such a registry and pass it as an argument to a processor, for which the behavior would be undefined.

This should be considered for the next update.

**Comment by sesuncedu** [comment]
It's the latter situation that I was thinking about (and ties in with issue #13, which I haven't filed yet :)

**Comment by sesuncedu** [comment]
After rereading bits of microdata-rdf and bits of microdata in a different order, I thought I had an answer (the definition of vocabulary used for the algorithm says no vocabulary URI may be a prefix of another). 
However, the registry section is not part of the algorithm section, which talks about URI prefixes, and which allows multiple vocabularies to have the equal prefix strings (which are improper prefixes of each other). 

I think it's safe to do longest match, since there are no cases where it matters atm, and there's no requirement to signal an error if the keys in the registry are vocabulary IRIs.

**Comment by gkellogg** [comment]
Agreed, but at this point, the document has been frozen for publication this Tuesday. However, I would support adding tests that check for properly finding longest-prefix first. The text should also be added to the spec post-publication, so that it will be included in the next revision. Hopefully, this will happen when `@itemprop-reverse` becomes part of Microdata.
